tssc/step_implementers/push_artifacts/maven.py

```diff
@@ -56,23 +60,29 @@
     push-artifacts:
     - implementer: Maven
       config:
-        url: http://artifactory.company.com/repo
+        maven-push-artifact-repo-id: internal-id-name
+        maven-push-artifact-repo-url: url-to server
 
 Example: Generated Maven Deploy (uses both step configuration and previous results)
 
     mvn
       deploy:deploy-file'
-      -Durl=url
+      -Durl=maven-push-artifact-repo-url
       -Dversion=generate-metadata.version
       -DgroupId=package.artifact.group-id
       -DartifactId=package.artifact.artifact-id
       -Dfile=package.artifact.path
       -Dpackaging=package.artifact.package-type
+      -DrepositoryId=maven-push-artifact-repo-id
```

Comment:
Should this example also display `-s settings.xml`?

Reply:
done